includes/Discovery.php

```diff
@@ -221,7 +221,7 @@ public function print_open_graph_metadata() {
 		$metadata = $this->get_open_graph_metadata();
 
 		foreach ( $metadata as $name => $value ) {
-			printf( '<meta property="%s" content="%s" />', esc_attr( $name ), esc_attr( $value ) );
+			printf( '<meta property="%s" content="%s" />', esc_attr( $name ), esc_attr( $value ) ); // phpcs:ignore WordPressVIPMinimum.Security.ProperEscapingFunction.notAttrEscAttr
```

Comment:
Hm, why didn't it pass here? How would it need to be then for this rule to pass?

Reply:
It is a new rule added in 2.3.0 coding standards. It expects `esc_attr` to be used as part of a html tag and not part of `printf`. Basically this is a false positive.

Reply:
Could the rules be adjusted to not give this false positive?

Reply:
Seems to be a known issue - https://github.com/Automattic/VIP-Coding-Standards/pull/668

Reply:
Ah, thanks for flagging, so it should get updated again once the fix gets released. Could we add this as a comment to the code (E.g. something like `/* Disable reason: known false positive bug, see https://github.com/Automattic/VIP-Coding-Standards/pull/668 */`)?

Reply:
It was merged in 2.3.1. I have updated now remove this. However, there are other new rules.